### DISCUSSION TITLE
New release for fixing skin names?

### DISCUSSION THREAD

**Discussion by daemyung** [zeux/meshoptimizer#506] [General]
Could you make a new release including [this](https://github.com/zeux/meshoptimizer/pull/496)?

**Comment by zeux** [comment]
You can grab pre-release binaries from https://github.com/zeux/meshoptimizer/actions/runs/3299430986. The next release is definitely not ready yet, a lot of work is planned over the next few months - I suspect it's going to happen in December or January.
